generated/niswitch/enums.py

```diff
@@ -7,11 +7,11 @@ class HandshakingInitiation(Enum):
     MEASUREMENT_DEVICE = 0
     '''
     The `niSwitch Initiate
-    Scan <switchviref.chm::/niSwitch_Initiate_Scan.html>`__ VI does not
+    Scan <switchviref.chm::/Initiate_Scan.html>`__ VI does not
```

Comment:
I'm not sure what to make of this. We shouldn't have CHM stuff here anymore right?

Reply:
I'm not sure what to do with this. It is in the source. I can work on finding it and fixing the link to point to ni.com/whatever instead?

Reply:
#751